cmd/kwild/config/genesis.go

```diff
@@ -69,76 +69,3 @@ func loadGenesisAndPrivateKey(autoGen bool, privKeyPath, rootDir string) (privKe
 	fmt.Printf("Generated new genesis file: %v\n", genFile)
 	return privKey, genesisCfg, nil
 }
-
```

Comment:
Removing this, as we decided to use genesis app hash as a way to enforce genesis state through snapshots.